src/definitions/v.json

```diff
@@ -0,0 +1,126 @@
+[
+    {
+      "display": "Vertical Align",
+      "description": "Sets the vertical alignment of an element",
+      "link": "https://www.w3schools.com/cssref/pr_pos_vertical-align.asp",
+      "propertyName": "vertical-align",
+      "valuetemplate": "{0}",
+      "categories": ["alignment", "layout", "position"],
+      "tags": "alignment, layout, position",
+      "controls": [
+        {
+          "type": "select",
+          "default": "baseline",
+          "label": "Value",
+          "options": [
+            {
+                "name": "Align to baseline of container",
+                "value": "baseline"
+            },
+            {
+                "name": "Length (Custom)",
+                "valuetemplate": "{0}{1}",
+                "value": {
+                  "controls": [
+                    {
+                      "type": "number",
+                      "label": "Amount",
+                      "default": "0px"
```

Comment:
as because this is a number type, the `default` value should also be a number so please change it to number